Module1/todo_app/data/view_model.py

```diff
@@ -0,0 +1,42 @@
+from todo_app.data.card_items import Item
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+
+
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def not_started_items(self):
+        output_list = []
+
+        for item in self._items:
+            if item.ListID == "64ef5905b30edf49e85fa346":
+                output_list.append(item)
+            
+        return output_list
```

Comment:
As with your Board ID, I'd recommend loading these list ids from environment variables, or this view model will still only work for one fixed Trello board